pallet/assets-ext/rpc/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022-2023 Futureverse Corporation Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// You may obtain a copy of the License at the root of this project source code
+
+//! Runtime API definition required by ASSETS-EXT RPC extensions.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::Codec;
+use pallet_assets_ext::Config;
+use seed_primitives::{AssetId, Balance};
+
+sp_api::decl_runtime_apis! {
+	/// The RPC API to interact with AssetExt module
+	pub trait AssetsExtApi<AccountId, T> where
+		AccountId: Codec,
+		T: Config,
+	{
```

Comment:
Do we need the constraint of `T` (it's not used)

```suggestion
	pub trait AssetsExtApi<AccountId> where
		AccountId: Codec,
	{
```

Reply:
Saw it in all the rpc runtime file's in DEX, also in NFT - https://github.com/futureversecom/trn-seed/blob/a6312f4e41ee0f3be44042fd38fa59673c74fbb0/pallet/nft/rpc/runtime-api/src/lib.rs#L29

Reply:
Maybe we don't need it here; worth a shot to reuce the trait bounds.
```suggestion
	pub trait AssetsExtApi<AccountId: Codec>{
```